pkg/mcs/scheduling/server/apis/v1/api.go

```diff
@@ -167,6 +169,13 @@ func (s *Service) RegisterOperatorsRouter() {
 	router.GET("/records", getOperatorRecords)
 }
 
+// RegisterRegionsRouter registers the router of the regions handler.
+func (s *Service) RegisterRegionsRouter() {
```

Comment:
Shall we move it to config router?

Reply:
ok, how about "/config/region/:id/label/:key" and "/config/region/:id/labels"

Reply:
I think ok until we find a better one.